style.css

```diff
@@ -0,0 +1,585 @@
+    margin: 20px;
+    font-weight: 400;
+    color: #272a31;
+    margin-bottom: 15px;
+  }
+
+  .dates {
+    font-weight: 700;
+    margin: 20px;
+  }
+
+  .menu {
+    cursor: pointer;
+    align-self: center;
+    display: none;
+  }
+
+  .bar {
+    width: 15px;
+    margin: 4px auto;
+    background-color: black;
+    height: 2px;
+    display: none;
+  }
+}
+
+.card-section {
+  background-color: #272a31;
+  margin: 0;
+  padding: 20px 0;
+}
+
+.card-main {
+  align-content: center;
+  color: #fff;
+  text-align: center;
+  margin: 0;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 6px;
+  justify-content: center;
+  margin-bottom: 10px;
+}
+
+.cd-card {
+  display: flex;
+  gap: 20px;
+  background: #6c757d;
+  width: 100%;
+  align-items: center;
+}
+
+.fa-solid {
+  font-size: 30px;
+  width: 10%;
+}
+
+.fa-brands {
+  font-size: 30px;
+  width: 10%;
+}
+
+.cards-hd {
+  color: #ec5242;
+  font-size: 15px;
+  width: 30%;
+}
+
+.cards-pr {
+  color: #fff;
+  font-size: 2px;
+  font-weight: 100;
+  width: 55%;
+}
+
+.main-btn-2022 {
+  text-align: center;
+}
+
+.card-btn {
+  color: #fff;
+  text-align: center;
+  display: none;
+}
+
+.btn-2022 {
+  background: #ec5242;
+  padding: 10px;
+}
+
+@media (min-width: 768px) {
+  .card-section {
+    margin: 0;
+  }
+
+  .cards {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .cd-card {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    width: 18%;
+    padding-top: 10px;
+  }
+
+  .btn-2022 {
+    display: none;
+  }
+}
+
+.speakers-section {
+  background-color: #d3d3d3;
+}
+
+.spks-main {
+  display: flex;
+  flex-direction: column;
+}
+
+.speakers {
+  display: flex;
+  margin: 0;
+  align-items: center;
+  padding: 0;
+  flex-direction: row;
+  gap: 14px;
+  width: 100%;
+}
+
+.speaker-pr {
+  margin: 0;
+  padding: 10px;
+}
+
+.h2-h2 {
+  font-size: 10px;
+}
+
+.h3-h3 {
+  font-size: 5px;
+  font-weight: 500;
+  color: #ec5242;
+}
+
+.p2-p3 {
+  font-size: 2px;
+  font-weight: 100;
+  color: #272a31;
+}
+
+.body-p2 {
+  width: 70%;
+}
+
+.img-tg {
+  width: 20%;
+}
+
+.img-tg img {
+  width: 100%;
+}
+
+.partners-section {
+  background-color: #272a31;
+  padding: 0;
+  margin: 0;
+}
+
+.h-ptns {
+  align-content: center;
+  color: #d3d3d3;
+  text-align: center;
+  margin: 0;
+}
+
+.ul-list {
+  display: flex;
+  list-style: none;
+  gap: 50px;
+  color: #d3d3d3;
+  justify-content: center;
+  text-align: center;
+  margin: 0;
+  padding: 10px;
+}
+
+.li-li {
+  padding: 5px;
+}
+
+.rights-section {
+  display: flex;
+  gap: 30px;
+  align-content: center;
+  background-color: #d3d3d3;
+  height: auto;
+  justify-content: center;
+}
+
+@media (min-width: 768px) {
+  .card-main {
+    padding: 20px 0;
+  }
+
+  .spks-main {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    justify-content: center;
+    width: 90%;
+    margin: 0 auto;
+    row-gap: 20px;
+  }
+}
+
+.about-section {
+  background: #d3d3d3;
+}
```

Comment:
- [ ] Please fix the spacing between the sections 
- [ ]  The bottom section images are organized in a vertical column in mobile version
![ScreenShot_20221126200302](https://user-images.githubusercontent.com/89518488/204100365-ce30d3cc-b058-4cfe-815e-f3191ed153ee.png)